sdk/android/analytics.md

```diff
@@ -658,8 +658,9 @@ public void addCustomEndpointAttribute() {
 #### Assigning User IDs to Endpoints
 Assign user IDs to endpoints by doing either of the following:
 
-Manage user sign-up and sign-in with Amazon Cognito user pools.
-Use the Amazon Pinpoint client to assign user IDs without using Amazon Cognito user pools.
+* Manage user sign-up and sign-in with Amazon Cognito user pools.
```

Comment:
Nit: These are fragments and don't need punctuation.

More than a nit: I'm not sure what these mean on their own. Where can the reader go to learn more?